src/main/java/com/microsoft/azuresamples/authenticationb2c/AuthHelper.java

```diff
@@ -138,6 +139,15 @@ private static void redirectToAuthorizationEndpoint(final HttpServletRequest req
         if (policy.equals(EDIT_PROFILE_POLICY) && msalAuth.getAuthenticated()) {
             parameters = AuthorizationRequestUrlParameters.builder(REDIRECT_URI, Collections.singleton(SCOPES))
                     .responseMode(ResponseMode.QUERY).state(state).nonce(nonce).build();
+        } else if (Config.getProperty("app.prompt") != null) {
+            if (!PROMPT.trim().equals("")) {
```

Comment:
```suggestion
        } else if (!PROMPT.trim().equals("")) {
```
Maybe we can just delete the null check and require the `aad.prompt` config. the config loader causes the program to exit if there is a null param anyhow.